functional-tests/fn_bridge_reassignment.py

```diff
@@ -0,0 +1,73 @@
+import time
+
+import flexitest
+
+import testenv
+from constants import UNSPENDABLE_ADDRESS
+from fn_bridge_withdraw_happy import (
+    DEPOSIT_AMOUNT,
+    check_initial_eth_balance,
+    deposit_twice,
+    do_withdrawal,
+    setup_services,
+)
```

Comment:
Maybe we should move these to utils?
What you think?
They are being used elsewhere than `fn_bridge_withdraw_happy`.